content/en/docs/concepts/workloads/controllers/job.md

```diff
@@ -289,6 +289,10 @@ starts a new Pod.  This means that your application needs to handle the case whe
 pod.  In particular, it needs to handle temporary files, locks, incomplete output and the like
 caused by previous runs.
 
+By default, each pod failure is counted towards the `.spec.backoffLimit` limit,
+see [pod backoff failure policy](#pod-backoff-failure-policy). However, you can
+customize this by setting the Job's [pod failure policy](#pod-failure-policy).
```

Comment:
```suggestion
customize pod failure limit by setting the Job's [pod failure policy](#pod-failure-policy).
```

Reply:
I think in this `this` means "handling of pod failures". Updated to make it clear.